VelociraptorPlatform-Electron/public/lib/tooltips.js

```
@@ -0,0 +1,826 @@
+ *   window.tooltipService.show(element, 'Custom tooltip', { position: 'bottom' });
+ *   window.tooltipService.hide();
+ *
+ * @module TooltipService
+ * @version 1.0.0
+ */
+
+class TooltipService {
+    constructor() {
+        // Singleton check
+        if (window.tooltipService) {
+            return window.tooltipService;
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)